tests/compliance/blackduck-spec.js

```diff
@@ -0,0 +1,176 @@
+'use strict'
+
+const rewire = require('rewire')
+const sinon = require('sinon')
+const expect = require('chai').expect
+const Promise = require('promise')
+const logger = require('../../lib/logger')
+const blackduck = rewire('../../lib/compliance/blackduck')
+
+describe('blackduck', () => {
+  let sandbox
+  beforeEach(() => {
```

Comment:
Looks like we've got a mix of passing in arrow functions and regular functions to things like `beforeEach()` and `it()` in this file. I would be consistent. As much as I hate to say it (because I'm lazy and generally use arrow functions whenever possible) it's actually considered bad practice to use arrow functions in `mocha` specs b/c of the `this` context that can be used (specifically to set the timeout): https://mochajs.org/#arrow-functions